CMakeLists.txt

```
@@ -502,6 +502,37 @@ endif()
 # initialize CLI11 submodule
 cpp_cc_git_submodule(CLI11 BUILD PACKAGE CLI11 REQUIRED)
 
+# =============================================================================
+# Enable NMODL code-generator support
+# =============================================================================
+if(NRN_ENABLE_CORENEURON)
+
+  set(NMODL_ENABLE_PYTHON_BINDINGS
+      OFF
+      CACHE BOOL "Enable NMODL python bindings")
+  nrn_add_external_project(nmodl DISABLE_ADD)
+  add_subdirectory(${PROJECT_SOURCE_DIR}/external/nmodl ${CMAKE_BINARY_DIR}/external/nmodl)
```

```suggestion
  nrn_add_external_project(nmodl)
```

IIRC, `DISABLE_ADD` was only required because we included NMODL in a subdirectory, with a build directory outside the location where it was included.

---

I remember trying, but it still seemed to be essential.

---

Seemed essential or failed to build?

Can always be simplified "later", too.

---

essential because it failed to build. "seemed" because the statement if from memory. i'll try once more.

---

which might be completely wrong, at least locally it today works without. let's see what ci thinks about the change.